### ISSUE TITLE
Toggle HUD elements command

### ISSUE THREAD

**Issue by cqb13** [MeteorDevelopment/meteor-client#3133]
### Describe the feature

Add a command similar to the toggle that allows the user to toggle specific hud elements. 

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by I-Got-You-Dead** [comment]
Wanna know something cool? There is a hidden Easter egg in meteor where if you go to that little bud tab at the top of the gui, you can edit your hud.

**Comment by cqb13** [comment]
Yes, but that takes time, and typing in chat is faster. Also this would allow you to make macros to toggle specific hud elements.

**Comment by I-Got-You-Dead** [comment]
Cope
